ingest/ingest-publications.py

```diff
@@ -44,6 +44,7 @@
 # standard filters
 non_empty_str = lambda s: True if s else False
 has_label = lambda o: True if o.label() else False
+print("PID: ", str(os.getpid))
```

Comment:
Is there a better way to handle logging in python? Something like log4j?

Reply:
I do handle other log info with the logger, these statements just go to stdout. I figured stdout should just provide a very basic status. Perhaps this is best in the logger since it's more of a debug. I'll move it.

Reply:
moved most prints to logging, prints are reserved now for basic screen output